src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -482,6 +484,10 @@ const OfflineTokenRow = ({ onValueChanged, validationFailed }) => {
             option: "default",
             message: _("Validating..."),
         },
+        LOADING: {
+            option: "default",
+            message: _("Checking for saved token..."),
```

Comment:
This doesn't involve the network or any heavy computation, right? Just checking localStorage. Is it really worth introducing a state and translatable string for this? How long do you actually see this? This should be a small fraction of a second.

If it does take longer than 0.5s for some reason, maybe a spinner is enough?

Reply:
It does invole network delay of checking validity of the token thru RHSM REST API. But the text will be changed to "Checking validity of saved token" with a spinner next to it as shown in the new mockup: https://github.com/cockpit-project/cockpit-machines/pull/774#issuecomment-1210577729